### PR TITLE
Jwineman/pi 552 advanced zone editor check

### PR DESCRIPTION
**Pull request by jwineman**
@thellimist

### CODE REVIEW

File: src/Cpanel/Zone/Partial.php [hunk 1]
```diff
@@ -16,6 +16,7 @@ class Partial
 
     const FORWARD_TO_SUFFIX = "cdn.cloudflare.net";
     const RESOLVE_TO_PREFIX = "cloudflare-resolve-to.";
+    const ADVANCED_ZONE_EDIT_DISABLED_ERROR = "CloudFlare cPanel Plugin configuration issue! Please contact your hosting provider to enable \"Advanced DNS Zone Editor\"";
```
Comment (thellimist) [inline src/Cpanel/Zone/Partial.php]: Localization?

Comment (jwineman) [inline src/Cpanel/Zone/Partial.php]: Backends aren't localized *yet*

File: src/Cpanel/CpanelAPI.php [hunk 2]
```diff
@@ -292,6 +292,6 @@ public function isAdvancedZoneEditEnabled() {
          * https://documentation.cpanel.net/display/SDK/Guide+to+the+LiveAPI+System+-+The+cpanelfeature%28%29+Method
          * Returns 1 = enabled, 0 = disabled
```
Comment (thellimist) [inline src/Cpanel/CpanelAPI.php]: Docs should be
Returns True = enable, False = disabled

Comment (jwineman) [inline src/Cpanel/CpanelAPI.php]: that comment is referring to what the cPanel method itself returns, I updated the PHPDoc on the method to reflect the fact that the method returns a Boolean though.

**Pull request merged**